.github/workflows/tests.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        platform: [ubuntu-20.04, ubuntu-22.04]
+        platform: [ubuntu-18.04, ubuntu-22.04]
```

Comment:
We clearly need to test this (as your PR description and the existence of this PR shows), but since GitHub have [deprecated 18.04](https://github.com/actions/runner-images/issues/6002) with the current target being fully unsupported next Saturday (1 Apr), we should probably start looking for other ways to do this (maybe even next sprint).

I've asked in ~github-actions-self-hosted-runners about whether we could get an 18.04 self-hosted runner, but otherwise we'll probably need to look into running the tests in LXD.

CC @sergiusens since this might become fairly urgent for us to do the migration work.

Reply:
oh man good call
let's talk about this soon